bot.go

```diff
@@ -302,11 +302,28 @@ func (bot *BotAPI) Send(c Chattable) (Message, error) {
 	}
 
 	var message Message
-	err = json.Unmarshal(resp.Result, &message)
+	resultStr := string(resp.Result)
+	if resultStr != "true" {
+		err = json.Unmarshal(resp.Result, &message)
+	}
 
 	return message, err
 }
 
+// helper to get raw params
+func (bot *BotAPI) PreviewParams(c Chattable) (map[string]string, error) {
+	paramsMap := make(map[string]string)
+	params, err := c.params()
+	if params == nil {
+		return paramsMap, err
+	}
+
+	for key, value := range params {
```

Comment:
You should be able to convert Params into a map without having to loop through it because it internally is a map.